UI/Tw.html

```diff
@@ -0,0 +1,496 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+      <meta name="keywords" content="HTML, BSU training, website">
+    <title>Thinhill</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+          <span>
+              <p> Thinhill</p>
+          </span>
+          <div class="hSp1">
+             Hello <br> Nadia!
+           </div>
+          <span>
+              <p><b><u><a href="out.html" target="_blank">Sign out</a></u></b></p>
+          </span>
+      </header>
+	  
+	  
+      <main>
+        <aside class="column right-aside">
+
+            <nav>
+                <button type = "submit" class="b1"><b>+ Add a review</b></button>
+            </nav>
+
+       <article>
+		   <hr>
+		   <p> <span class="colortext"> Sale 10% </span> </p>
+		   <p> <img src="6.jpg"  width= "170" height= "170" class="leftimg" >
+           <p class="name"><b>VINTAGE BAG</b></p>
+		   <p class="text1">Louis Vuitton</p>
+		   
+           <p>28.02.2021 - 10.04.2021</p>	   
+		   <p class="container1">Short description</p>
+		   <div>
+			  <p><b><u><a href="full.html" target="_blank">Full description</a></u></b></p>
+           </div>
+		   </p>
+           <p>#Vintage bag</p>
+		   
+		  
+		   <div style="clear: both"></div>
```

Comment:
Please don't use inline styles. If you need to clean floats, you can for example add `class="clearfix"` and define styles in css file.